src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -208,6 +209,7 @@ public void robotPeriodic() {
     public void autonomousInit() {
 
         if (subsystems.drivebaseSubsystem != null) {
+            subsystems.drivebaseSubsystem.resetGyroAngle(Rotation2.ZERO);
```

Comment:
Please remove this in case we want to have an autonomous mode where the robot does not start at an angle of zero.